examples/hello-world/hello.py

```diff
@@ -9,9 +9,10 @@
 
 async def worker(context: WorkContext, tasks: AsyncIterable[Task]):
     async for task in tasks:
-        context.run("/bin/sh", "-c", "date")
+        script = context.new_script()
+        script.run("/bin/sh", "-c", "date")
 
-        future_results = yield context.commit()
+        future_results = yield script
         results = await future_results
         task.accept_result(result=results[-1])
```

Comment:
Since we're already updating this example, why don't we use the new way of obtaining results?
```
future_result = script.run("/bin/sh", "-c", "date")

yield script

result = await future_result
task.accept_result(result=result)
```
Or something along these lines.

Reply:
I encountered some issues with the new approach and didn't want to distract myself further _yet_ ...

Reply:
Fair enough, let's move this to a separate PR then.